hr_attendance_report_theoretical_time/readme/CONFIGURE.rst

```diff
@@ -1,4 +1,4 @@
-You need to be at least "Attendance / Manual Attendance" for being able to see
+You need to be at least "Attendance / Officer" for being able to see
```

Comment:
@jarroyomorales employees must see the report to check if they have done more o less hours

Reply:
Okay, I changed it again